packages/core/src/renderers/utils/QuadUv.js

```diff
@@ -0,0 +1,103 @@
+import Geometry from '../../geometry/Geometry';
+import Buffer from '../../geometry/Buffer';
+
+/**
+ * Helper class to create a quad with uvs like in v4
+ *
+ * @class
+ * @memberof PIXI
```

Comment:
This should probably be `@private`?

Reply:
I dont know what advantage has v5 Quad against v4 Quad so i want them both present.